src/meta/api/src/schema_api_test_suite.rs

```diff
@@ -1290,6 +1299,36 @@ impl SchemaApiTestSuite {
         Ok(())
     }
 
+    /// TODO: we need to implement get and drop later.
+    #[tracing::instrument(level = "debug", skip_all)]
+    async fn catalog_create_get_drop<MT: SchemaApi>(&self, mt: &MT) -> anyhow::Result<()> {
+        let tenant = "tenant1";
+        let catalog_name = "catalog1";
+
+        info!("--- create catalog1");
+        let req = CreateCatalogReq {
+            if_not_exists: false,
+            name_ident: CatalogNameIdent {
+                tenant: tenant.to_string(),
+                catalog_name: catalog_name.to_string(),
+            },
+            meta: CatalogMeta {
+                catalog_option: CatalogOption::Iceberg(IcebergCatalogOption {
+                    storage_params: Box::new(StorageParams::Azblob(StorageAzblobConfig {
+                        container: "container".to_string(),
+                        ..Default::default()
+                    })),
+                }),
+                created_on: Utc::now(),
+            },
+        };
+
+        let res = mt.create_catalog(req).await?;
+        info!("create catalog res: {:?}", res);
```

Comment:
It would be better if a `get_catalog()` method was available to enable checking of the `create_catalog` result.

Reply:
added.